lib/pseudo_date/parser.rb

```diff
@@ -86,8 +86,8 @@ def self.parse_string(input)
       year = date_array.select{ |part| part.length == 4 }.first
       unless year.nil? || date_array.length != 3
         if date_array.first == year
-          month = date_array.last
-          day = date_array[1]
+          month = date_array[1]
+          day = date_array.last
```

Comment:
Pretty sure this is going to break dates that come in like `01/04/04`.

Reply:
I'm not sure it will. Where were dates in that format being handled before?